### PR TITLE
INFRA-2360 Move account app to the AKS cluster

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

File: charts/accountapp/templates/tests/test-connection.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  labels:
+{{ include "accountapp.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args:  ['{{ include "accountapp.fullname" . }}:8080/labels']
```
Comment (olblak) [inline charts/accountapp/templates/tests/test-connection.yaml]: Is `/labels` supposed to work? Shouldn't it be `/login`?

File: charts/accountapp/templates/service.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  type: {{ .Values.service.type }}
+  ports:
+    - name: frontend
+      protocol: TCP
+      targetPort: frontend
+      port: 5000
+    - name: backend
+      protocol: TCP
+      targetPort: backend
+      port: 8080
```
Comment (olblak) [inline charts/accountapp/templates/service.yaml]: Fronted and backend are provided by the same docker image that listen on port 8080

File: config/default/accountapp.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+ingress:
+  enabled: true
+  annotations:
+    "cert-manager.io/cluster-issuer": "letsencrypt-prod"
+    "kubernetes.io/ingress.class": "public-ingress"
+    "nginx.ingress.kubernetes.io/hsts": "true"
+    "nginx.ingress.kubernetes.io/ssl-redirect": "true"
+
```
Comment (olblak) [inline config/default/accountapp.yaml]: Each container will generate a cookie to store the access, so it's important that a user always sticks to the same container which is why 
`nginx.ingress.kubernetes.io/affinity: cookie` is needed when having multiple replicas

File: charts/accountapp/templates/service.yaml (hunk 2 above)
Comment (timja) [reply charts/accountapp/templates/service.yaml]: 🤕

File: config/default/accountapp.yaml (hunk 3 above)
Comment (timja) [reply config/default/accountapp.yaml]: why is it stateful 😢 

fixed..

File: charts/plugin-site/templates/tests/test-connection.yaml [hunk 4]
```diff
@@ -11,5 +11,5 @@ spec:
     - name: wget
       image: busybox
       command: ['wget']
-      args:  ['{{ include "plugin-site.fullname" . }}:8080/labels']
+      args:  ['{{ include "plugin-site.fullname" . }}:8080/login']
```
Comment (timja) [inline charts/plugin-site/templates/tests/test-connection.yaml]:
```suggestion
      args:  ['{{ include "plugin-site.fullname" . }}:8080/labels']
```

File: charts/plugin-site/templates/service.yaml [hunk 5]
```diff
@@ -9,7 +9,7 @@ spec:
     - name: frontend
       protocol: TCP
       targetPort: frontend
-      port: 5000
+      port: 5000
```
Comment (timja) [inline charts/plugin-site/templates/service.yaml]:
```suggestion
      port: 5000
```

File: charts/accountapp/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for plugins.jenkins.io
```
Comment (halkeye) [inline charts/accountapp/Chart.yaml]: lies! accountapp (i assume) isn't plugins.jenkins.io

Comment (timja) [reply charts/accountapp/Chart.yaml]:
```suggestion
description: A Helm chart for accounts.jenkins.io
```

Comment (timja) [reply charts/accountapp/Chart.yaml]: :shipit:

**Comment by olblak** [comment]
The secrets file is ready, so this pr can be merge once the maintainer label is "fixed"

File: charts/accountapp/templates/_helpers.tpl [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+*/}}
+{{- define "accountapp.labels" -}}
+app.kubernetes.io/name: {{ include "accountapp.name" . }}
+helm.sh/chart: {{ include "accountapp.chart" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+jenkins.io/maintainer: {{ range $maintainer := .Chart.Maintainers }}{{ $maintainer.Name }} {{ end }}
```
[the remark this answers is not in the document]
Comment (timja) [reply charts/accountapp/templates/_helpers.tpl]: suggestions on github make that easier 😉 
```suggestion
jenkins.io/maintainer: {{ (index .Chart.Maintainers 0).Name }}
```

**Pull request merged**